Dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM openshift/origin-base
+FROM openshift/origin-release:golang-1.7
 MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
 
-RUN yum install -y golang openscap-scanner && yum clean all
```

Comment:
@enoodle @ilackarms @pweil- hey! :blush: 

We haven't caught this... how do we plan to have openscap working without installing it? :-1: